app/src/main/java/com/nearbynest/app/ui/list/ListFragment.kt

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright COPYRIGHT
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nearbynest.app.ui.list
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.nearbynest.app.R
+
+class ListFragment : Fragment() {
```

Comment:
Use `BaseFragment` and `ListFragment` is not the best name.

Reply:
Made necessary changes which uses BaseFragment in respective Fragment.